tasks/pipeline.py

```diff
@@ -528,6 +528,20 @@ def changelog(ctx, new_commit_sha):
     if not old_commit_sha:
         print("Old commit sha not found, exiting")
         return
+
+    commit_range_link = f"https://github.com/DataDog/datadog-agent/compare/{old_commit_sha}..{new_commit_sha}"
+    empty_changelog_msg = "No new System Probe related commits in this release :cricket:"
+    slack_message = (
+        "The nightly deployment is rolling out to Staging :siren: \n"
+        + f"Changelog for <{commit_range_link}|commit range>: `{old_commit_sha}` to `{new_commit_sha}`:\n"
+    )
+
+    if old_commit_sha == new_commit_sha:
+        print("No new commits found, exiting")
+        slack_message += empty_changelog_msg
```

Comment:
wouldn't it be clearer if this message explicitely mentionned that there's no new commit at all?